### PR TITLE
Various adjustments to validation interface

### PR DESCRIPTION
**Pull request by ivansg44**

### CODE REVIEW

**Comment by ivansg44** [comment]
image

image

File: geem/templates/geem/portal.html [hunk 1]
```diff
@@ -261,6 +261,10 @@ <h3 style="color:white;padding-top:10px">Genomic Epidemiology Entity Mart Protot
 
             <div id="user_validation_grid" class="ag-theme-balham"></div>
             <div id="ontology_validation_grid" class="ag-theme-balham-dark"></div>
+            <div class="infoBox" id="validation_info_box">
+              Select a specification resource to load a grid containing
```
[the remark this answers is not in the document]
Comment (ddooley) [reply geem/templates/geem/portal.html]: Also, maybe make the ontology view grid a light grey, a little less extreme than goth black.

[the remark this answers is not in the document]
Comment (ivansg44) [reply geem/templates/geem/portal.html]: They only have two built-in themes (light and dark), and the only easy way to customize them is using sass variables. Are you okay with just making both grids light coloured?

[the remark this answers is not in the document]
Comment (ddooley) [reply geem/templates/geem/portal.html]: Sure!

**Pull request merged**